R/fftrees_threshold_factor_grid.R

```diff
@@ -69,7 +69,7 @@ fftrees_threshold_factor_grid <- function(thresholds = NULL,
     results$threshold <- thresholds
 
     # Add accuracy statistics:
-    results <- cbind(results, Add_Stats(results,
+    results <- cbind(results, add_stats(results,
```

Comment:
Good fix here!

Reply:
Thanks — it did look very odd when looking at the package index (and I was wondering whether I missed some special significance here...)